test/test_output.py

```diff
@@ -0,0 +1,221 @@
+        )
+
+    def test_output_schema_log_json_default(self):
+        self.library.output_schema(self.output_dict)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=True
+        )
+
+    def test_output_schema_log_json_console_true(self):
+        self.library.output_schema(self.output_dict, also_console=True)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=True
+        )
+
+    def test_output_schema_log_json_console_false(self):
+        self.library.output_schema(self.output_dict, also_console=False)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=False
+        )
+
+    def test_output_schema_log_json_sort_keys_false(self):
+        self.library.output_schema(self.output_dict, sort_keys=False)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=True
+        )
+
+    def test_output_schema_log_json_sort_keys_true(self):
+        self.library.output_schema(self.output_dict, sort_keys=True)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=True, also_console=True
+        )
+
+    def test_output_schema_log_json_console_true(self):
+        self.library.output_schema(self.output_dict, also_console=True)
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=True
+        )
+
+    def test_output_schema_string_values_true(self):
+        self.library.output_schema(self.output_dict, also_console="true")
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=True
+        )
+
+    def test_output_schema_string_values_false(self):
+        self.library.output_schema(self.output_dict, also_console="false")
+        self.library.log_json.assert_called_with(
+            self.output_schema_dict, sort_keys=False, also_console=False
+        )
+
+
+class TestOutputConsole(unittest.TestCase):
+    def setUp(self) -> None:
+        self.output_dict = {"robotframework": "cool"}
+        self.output_schema_dict = {
+            "properties": {"robotframework": {"type": "string"}},
+            "required": ["robotframework"],
+            "type": "object",
+        }
+        self.output_console = '\n{\n    "robotframework": "cool"\n}\n'
+        self.output_schema_console = (
+            "\n"
+            + "{\n"
+            + '    "type": "object",\n'
+            + '    "properties": {\n'
+            + '        "robotframework": {\n'
+            '            "type": "string"\n'
+            "        }\n"
+            "    },\n"
+            + '    "required": [\n'
+            + '        "robotframework"\n'
+            + "    ]\n"
+            + "}\n"
```

Comment:
This looks a bit weird with missing `+` and mixed quotes (`'` and `"`). Could you change this to a string block (`"""`)?

Reply:
Actually... Is this even needed?

Reply:
Tried this locally and removing the `self.output_schema_console` didn't affect to test outcome:
```
$ nox -s test -- test/test_output.py 
nox > Running session test
nox > Creating virtual environment (venv) using python3.8 in .venv/test
nox > python -m pip install --upgrade -r requirements-dev.txt
nox > pre-commit install
pre-commit installed at .git/hooks/pre-commit
nox > python -m unittest discover
....

Property 'abba' does not exist in:
{
    "schema": {
        "properties": "abba"
    }
}
.Unrecognized log level 'NOT ACCEPTABLE'. Using default log level 'WARN'.
.....................
----------------------------------------------------------------------
Ran 26 tests in 0.110s

OK
nox > pytest test/test_output.py
============================= test session starts ==============================
platform darwin -- Python 3.8.9, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /Users/jjauhiainen/projects/Eficode/oss/RESTKivipe
plugins: clarity-1.0.1, instafail-0.4.2, spec-3.2.0
collected 19 items                                                             

test/test_output.py ...................                                  [100%]

============================== 19 passed in 0.11s ==============================
nox > Session test was successful.
```